### ISSUE TITLE
Package HomeCare & Disaster Relief Survey metadata via Metecho

### ISSUE THREAD

**Issue by kdveverka** [SFDO-Community-Sprints/GrassrootsMobileSurveyApp#54]

**Comment by shunkosa** [comment]
Thanks, it's a child task of #44

**Comment by kdveverka** [comment]
> Thanks, it's a child task of #44

Shun - Mind if I make a Milestone for the demo org? Can't really associate the issues otherwise it looks like.

**Comment by shunkosa** [comment]
@kdveverka Of course! Thank you for organizing🙏

**Comment by johncastelloe** [comment]
I added the remaining homecare fields to a dev scratch org in Metecho and tried to retrieve the changes (everything but the profiles), and I got the error in this screenshot. Any ideas what the obstacle might be?

Here's the commit message that I was going to include:
- Added all homecare survey fields that don't overlap with disaster relief survey. Updated some field names, help text, and descriptions. I tried to make the field names as short as possible and tailor the help text to resemble how a field worker might ask the question. Original field names in the spreadsheet are included in description.

![metecho-retrieve-changes-error](https://user-images.githubusercontent.com/13836716/137610909-62d3bdef-09d4-4cf3-9bbb-f2416e078f12.png)

**Comment by shunkosa** [comment]
First version of HomeCare survey metadata is merged at https://github.com/SFDO-Community-Sprints/GrassrootsSurveySalesforcePackage/pull/7

**Comment by johncastelloe** [comment]
@eehjunggnujhee, I have a task from our last meeting to check in with you about this, so here I am :). I helped do some QA testing in our October sprint, using the [Survey Fields](https://docs.google.com/spreadsheets/d/1U1qbfa-fL87TVlT2ssjURv9tM53fzEMJW1MG-WWSFBY/edit#gid=0) spreadsheet. More recently I added a lot of notes to the [User Stories](https://docs.google.com/spreadsheets/d/1vAe-aENiSn-NEQrVfbG0CedRNMwPqvXNQlAzcD75pSY/edit#gid=0) spreadsheet for Home Care; those are probably most relevant to the core design of the app, but maybe some aspects are relevant for our sample survey package too.

**Comment by eehjunggnujhee** [comment]
Hi @johncastelloe  Happy New Year! All. I reviewed the last meeting notes and saw we are waiting for metadata updates on the core package. We can probably meet sometime after we hear from Trish and Shun?? @travelertrish @shunkosa  How did the meeting went? Let us know! @kdveverka thanks for the amazing PM work.

**Comment by eehjunggnujhee** [comment]
In fact, @johncastelloe  I can actually create Data Dictionary for these sample surveys. :D 
Let me see if I can spin up another metech and extract metadata from all the new fields.

**Comment by kdveverka** [comment]
Thanks John & Jung!

**Comment by eehjunggnujhee** [comment]
Hi @johncastelloe  I see that you already made changes in metecho dev org. Any name changes in API names will create new metadata. Any old one needs to be manually removed directly from a branch repository. Let's review when this becomes an issue. Thank you

**Comment by johncastelloe** [comment]
Happy New Year to you too, @eehjunggnujhee! Any changes in the metecho scratch org I created about a month ago can be ignored; I don't think I did very much before Trish and I decided to revisit the data model with the group first. @travelertrish, maybe we can have a check-in about this at our planned meeting time in about 30 minutes. I'm having trouble remembering how we left things. Anyone else is welcome to participate too of course (@eehjunggnujhee, @kdveverka, etc.).
